app/org/sagebionetworks/bridge/dynamodb/DynamoCompoundActivityDefinition.java

```diff
@@ -0,0 +1,116 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.util.List;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTypeConverted;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBVersionAttribute;
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.google.common.collect.ImmutableList;
+
+import org.sagebionetworks.bridge.models.schedules.CompoundActivityDefinition;
+import org.sagebionetworks.bridge.models.schedules.SchemaReference;
+import org.sagebionetworks.bridge.models.schedules.SurveyReference;
+
+/**
+ * DynamoDB implementation of CompoundActivityDefinition. The hash key is study ID and the range key is the task ID.
+ * This allows us to namespace compound activity definitions to studies and to list all definitions in a study.
+ */
+@DynamoDBTable(tableName = "CompoundActivityDefinition")
+@JsonFilter("filter")
+public class DynamoCompoundActivityDefinition implements CompoundActivityDefinition {
+    private List<SchemaReference> schemaList = ImmutableList.of();
+    private String studyId;
+    private List<SurveyReference> surveyList = ImmutableList.of();
+    private String taskId;
+    private Long version;
```

Comment:
These are going to need a name or something to display them in a selector (a list or pulldown or something). 

I'm not sure what taskId is... is this the compoundActivityDefinitionId or Guid? If these are only created when we schedule tasks (and that's the task's ID), how will people define these beforehand to include in schedules?